examples/kitchen-sink-example/pages/api/shows.ts

```diff
@@ -7,14 +7,18 @@ export default withApiAuthRequired(async function shows(req, res) {
       scopes: ['read:shows']
     });
 
-    const response = await fetch('http://localhost:3001/api/my/shows', {
+    const baseURL = process.env.AUTH0_BASE_URL?.indexOf('http') === 0 ? 
+      process.env.AUTH0_BASE_URL : 
+      `https://${process.env.AUTH0_BASE_URL}`;
+
+    const response = await fetch(baseURL + 'api/my/shows', {
```

Comment:
It looks like we can not use relative URLs here, so I added the base_url. Is that expected?

Reply:
Also, Next recommends to not use fetch when calling an API from another API, but I am not sure I understand how we add the access tokens in that case.

https://nextjs.org/docs/basic-features/data-fetching

> Note: You should not use fetch() to call an API route in your application. Instead, directly import the API route and call its function yourself. You may need to slightly refactor your code for this approach.
> 
> Fetching from an external API is fine!

Reply:
Because this is a contrived example (`api/my/shows` would normally be on am external domain) - this is ok (have added a comment to the code)